ModelBundle/Repository/NodeRepository.php

```diff
@@ -302,9 +302,9 @@ protected function getTreeParentIdLevelAndLanguage($parentId, $nbLevel, $languag
     protected function findLastVersion(Stage $qa)
     {
         $elementName = 'node';
+        $qa->sort(array('version' => 1));
```

Comment:
this one has been added in the pr : https://github.com/open-orchestra/open-orchestra-model-bundle/pull/448